### PR TITLE
feature/AT-8025-page-3-travel-estimates

### PR DESCRIPTION
**Pull request by tonysumner-ccpo**

### CODE REVIEW

File: src/sass/utils/base.scss [hunk 1]
```diff
@@ -47,6 +47,10 @@ body {
   border: 1px solid !important
 }
 
+.no-border {
```
Comment (ericyoungquist-ccpo) [inline src/sass/utils/base.scss:50]: this class is already defined in `utils.scss`

File: src/steps/10-FinancialDetails/IGCE/TravelEstimates.vue [hunk 2]
```diff
@@ -1,15 +1,129 @@
...
+                width="190"
+                class="mr-2"
+                :rules="[$validators.required('', true)]"
+              />
+            </template>
+            <template v-if="travelFormFields === 'multiple'">
+              <div
+                v-for="(period, idx) in periods"
+                :key="idx"
+                :class="[(idx < periods.length-1 ? 'pb-5': ''), ' d-flex align-center']"
```
Comment (ericyoungquist-ccpo) [inline src/steps/10-FinancialDetails/IGCE/TravelEstimates.vue:47]: pls add class `pl-2` to the non-conditional set of classes - it's a little tight to the left 4px border

File: src/steps/10-FinancialDetails/IGCE/TravelEstimates.vue [hunk 3]
```diff
@@ -1,15 +1,129 @@
...
+                style="border-left: #544496 4px solid;"
+              >
+                <div class="text-right" style="width: 75px">
+                  {{ getOption(idx) }}
+                </div>
```
Comment (ericyoungquist-ccpo) [inline src/steps/10-FinancialDetails/IGCE/TravelEstimates.vue:52]: when errored, the base/option field labels should stay inline with the input. They're shifting down.
Screen Shot 2022-10-06 at 9 49 27 AM

File: src/steps/10-FinancialDetails/IGCE/TravelEstimates.vue [hunk 4]
```diff
@@ -1,15 +1,129 @@
...
-  <div>
-    Future page for Travel Estimates
-  </div>
+  <v-container fluid class="container-max-width">
+    <v-row>
+      <v-col class="col-12">
+        <h1 class="page-header">
+          How do you want to estimate your travel needs?
+        </h1>
+        <p>
```
Comment (ericyoungquist-ccpo) [inline src/steps/10-FinancialDetails/IGCE/TravelEstimates.vue:8]: please add class `page-intro` to this `p` tag to reduce the space between header and text

**Review by ericyoungquist-ccpo — CHANGES_REQUESTED** [review]

File: src/steps/10-FinancialDetails/IGCE/TravelEstimates.vue (hunk 2 above)
Comment (tonysumner-ccpo) [reply src/steps/10-FinancialDetails/IGCE/TravelEstimates.vue:47]: done

File: src/steps/10-FinancialDetails/IGCE/TravelEstimates.vue (hunk 3 above)
Comment (ericyoungquist-ccpo) [reply src/steps/10-FinancialDetails/IGCE/TravelEstimates.vue:52]: can you pls change `my-3` to `mt-2` ? it's sitting a little low now in relation to the input

File: src/steps/10-FinancialDetails/IGCE/TravelEstimates.vue [hunk 5]
```diff
@@ -1,15 +1,129 @@
...
+            :items="travelEstimateOptions"
+            @click="travelFormFields"
+            :value.sync="selectedTravelEstimate"
+            :rules="[$validators.required('Please select an option')]"
+          />
+        </div>
+
+      
+         <fieldset class="no-border" v-if="travelFormFields !==''">
+            <legend class="mb-4"><strong>Estimated travel costs per period</strong></legend>
```
Comment (ericyoungquist-ccpo) [inline src/steps/10-FinancialDetails/IGCE/TravelEstimates.vue:29]: I should have noticed this before too, sorry... when single input, the label "Estimated travel costs..." is normal weight. When entering for each period, it goes to `font-weight-500` not `<strong>` which is 700
Screen Shot 2022-10-06 at 10 41 24 AM

File: src/steps/10-FinancialDetails/IGCE/TravelEstimates.vue [hunk 6]
```diff
@@ -1,15 +1,129 @@
...
+            id="TravelEstimates"
+            :card="true"
+            :items="travelEstimateOptions"
+            @click="travelFormFields"
+            :value.sync="selectedTravelEstimate"
+            :rules="[$validators.required('Please select an option')]"
+          />
+        </div>
+
+
```
Comment (ericyoungquist-ccpo) [inline src/steps/10-FinancialDetails/IGCE/TravelEstimates.vue:27]: Missed this the first time around - the `hr` is missing between the radio group and inputs
Screen Shot 2022-10-06 at 10 39 56 AM

**Review by ericyoungquist-ccpo — CHANGES_REQUESTED** [review]

**Pull request merged**